### ISSUE TITLE
Distance metric for fbb setting

### ISSUE THREAD

**Issue by SCccc21** [DingfanChen/GAN-Leaks#1]
Hi there,

I read your paper, it is a very interesting work, and I really appreciate that you release the code. But it seems that the distance metric used in the code of fbb setting (Minkowski in sklearn, which is an image-level distance) is not consistent with the one in the paper (image_level_l2 + lambda2 * lpips). Which one shall I use? I tried to reimplement the experiment on cifar10, but the results obtained using the default metrics in your code are quite poor, which only achieve aucroc around 0.51 with size of dataset = 1000.

**Comment by DingfanChen** [comment]
Hi,
Thanks for your interest.
By default, lambda2=0 for the fbb setting (See Appendix B.1 of the paper). 
I haven't conducted experiments on the CIFAR10 dataset, so it would be hard for me to make conjectures/draw conclusions. There should be multiple factors that could affect the attack performance, e.g., the expected distance between samples within the dataset and the quality of the GAN models. If you are using our implementation for the victim GAN models, we would suggest readjusting the hyperparameters, as they are currently tuned for the face images.

**Comment by SCccc21** [comment]
Thank you for your reply! I didn't use your implementation for training the victim GAN. But I ll probably switch to CelebA dataset and check the results. Thanks!
